File: 1858.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');
let n = parseInt(lines.shift());
let v = lines.shift().split(" ");
let posicao = 0; let menor = parseInt(v[0]);
for(let i = 1; i < n; i++){
    if(v[i] < menor){
        menor = v[i];
        posicao = i;
    }
}
console.log(posicao+1);

File: 1858.cpp
#include <iostream>
using namespace std;

int main(){
    int numero_casos, menor, numero, posicao=1;
    cin >> numero_casos;
    cin >> menor;
    for(int i = 1; i<numero_casos; i++){
        cin >> numero;
        if(numero<menor){
            posicao=i+1;
            menor = numero;
        }
        
    }
    cout << posicao << endl;
}
